elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/ElideConfigParser.java

```diff
@@ -56,7 +56,8 @@ private void parse(String localConfigPath) throws Exception {
 
         //security
         String securityJson = util.getJsonConfig(localConfigPath, DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY).get(0);
-        populatePojo(securityJson, DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY);
+        populatePojo(util.resolveVariables(securityJson, this.variables),
+                DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY);
 
         //table
         populateTablesPojo(util.getJsonConfig(localConfigPath, DynamicConfigHelpersUtil.SCHEMA_TYPE_TABLE));
```

Comment:
util.resolveVariables should also be called in populateTablesPojo.

Reply:
populateTablesPojo method is calling util.resolveVariables